docs/user-guide.md

```diff
@@ -401,3 +401,7 @@ If the Docker Compose file has a volume specified for a service, the Deployment
 If the Docker Compose file has service name with `_` in it (eg.`web_service`), then it will be replaced by `-` and the service name will be renamed accordingly (eg.`web-service`). Kompose does this because "Kubernetes" doesn't allow `_` in object name.
 
 Please note that changing service name might break some `docker-compose` files.
+
+#### Warning about Labels
+
+`kompose.service.type` label should be defined with `ports` only, otherwise `kompose` will fail.
```

Comment:
This should be added under the `labels` section, not at the end of the user-guide file, otherwise, PR LGTM.